kustomize.go

```diff
@@ -153,7 +154,7 @@ func (r *Runner) KustomizeBuild(srcDir string, tempDir string, opts ...Kustomize
 	if err != nil {
 		return "", err
 	}
-	kustomizeArgs = append(kustomizeArgs, f)
+	kustomizeArgs = append(kustomizeArgs, f, "--enable-helm", "--helm-command="+u.HelmBinary)
```

Comment:
what will happen if u.HelmBinary is empty?

Reply:
Then `must specify --helm-command` is occurred.
Fixed it! （If u.HelmBinary is empty, `must specify --enable-helm` would be occurred.）